bp_fe/syn/Makefile

```diff
@@ -14,6 +14,7 @@ export TOUCH_PATH   := $(BP_FE_DIR)/syn/touchfiles
 
 export TB    ?= bp_fe_icache
 export CFG   ?= e_bp_unicore_half_cfg
+export DRAM  ?= dramsim3
```

Comment:
Maybe the default should be the AXI for faster regressions?

Reply:
It needs to support preloading, so if we can get the axi preload working then yeah I agree